DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/CourseSetupController.cs

```diff
@@ -94,5 +113,363 @@ public IActionResult AllCourseStatistics()
             var model = new AllCourseStatisticsViewModel(centreCourses, categories, topics);
             return View(model);
         }
+
+        [HttpGet]

```

Comment:
Fyi, when using http methods and routes you can merge them and define this in one line, i.e.
`[HttpGet("AddCourseNew")]`

Reply:
Oh cool, thanks. Implemented that in this controller